### PR TITLE
New kitchen recipes Part 1

### PR DESCRIPTION
**Pull request by Pyko1**
:cl: Pyko
add: assburgers. - Credit to Arzion for the original assburger icon on which I made minor edits.
add: asspie.
add: rat-kebab and double rat-kebab.
add: haggis, chunk of haggis,  haggis with neeps and tatties. - Credits to supermichael777 for icons.
add: Corn-Potato Pizza, credits to carbonhell for baking this monster IRL.
add: Hearthy Toast
add: Leg Taco
add: Butt-kebab
/:cl:
capture

Credit to Arzion for the original assburger icon on which I made minor edits.
Credits to supermichael777 for haggis icons.

### CODE REVIEW

**Comment by McBawbaggings** [comment]
should be buttburger but the description should be assburgers related imo

**Comment by Pyko1** [comment]
actually i had a hard time choosing from assburgers, buttburgers and a bumburger.

**Comment by Pyko1** [comment]
+	name = "assburger"
 +	desc = "What the hell, thats not an domesticated donkey meat, its a literal buttburger!"

**Comment by Pyko1** [comment]
Ultimately i chose the pun of having assburgers

**Comment by Pyko1** [comment]
prolly should make blue and xenobutt variants tho

**Comment by Pyko1** [comment]
didint really seem to do that much after i ate a whole burger on test server the od is 30 and the 10 is gone pretty fast.

**Comment by Pyko1** [comment]
at 5 units all it did was a single fart

**Comment by dreamcrab** [comment]
if it isnt called an assburger youre seriously fucking up

**Comment by Pyko1** [comment]
added asspie

**Comment by Pyko1** [comment]
need more food ideas preferably organ releated

**Comment by tortellinitony** [comment]
#4826 - Based on the previous ruling

**Comment by Pyko1** [comment]
There's no curses used, if you read the desc its clearly a reference to domesticated donkey.

File: hippiestation/code/modules/food_and_drinks/food/snacks_meat.dm [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+	w_class = WEIGHT_CLASS_NORMAL
+	list_reagents = list("nutriment" = 6, "vitamin" = 2)
+	tastes = list("rat meat" = 1, "metal" = 1)
+	foodtype = MEAT | GROSS
+
+/obj/item/reagent_containers/food/snacks/kebab/rat/double
+	name = "double rat-kebab"
+	icon_state = "doubleratkebab"
+	tastes = list("rat meat" = 2, "metal" = 1)
+	bonus_reagents = list("nutriment" = 6, "vitamin" = 2)
```
Comment (JohnGinnane) [inline hippiestation/code/modules/food_and_drinks/food/snacks_meat.dm]: Shouldn't this have more nutriment and/or vitamins?

Comment (Pyko1) [reply hippiestation/code/modules/food_and_drinks/food/snacks_meat.dm]: its bonus reagents applied in top of the base reagents

Comment (JohnGinnane) [reply hippiestation/code/modules/food_and_drinks/food/snacks_meat.dm]: Oh right, my 🐝

**Comment by Pyko1** [comment]
added haggish, still missing icons

File: hippiestation/code/modules/food_and_drinks/food/snacks_bread.dm [hunk 2]
```diff
@@ -0,0 +1,17 @@
+/obj/item/reagent_containers/food/snacks/store/bread/haggis
+	name = "haggis"
+	desc = "a savoury pudding containing intestines."
```
Comment (ghost) [inline hippiestation/code/modules/food_and_drinks/food/snacks_bread.dm]: Capitialise this

**Comment by dreamcrab** [comment]
yassss +58

**Comment by Pyko1** [comment]
haggis

**Comment by PopNotes** [comment]
these sprites make me want to kill myself

**Comment by Pyko1** [comment]
welp get me better ones then

**Comment by Pyko1** [comment]
Eww, corn-potato pizza added.

**Comment by Pyko1** [comment]
added butt-kebab

**Comment by Pyko1** [comment]
wee elp! part 1 done gross food added

**Comment by banthebantz** [comment]
+1

File: hippiestation/code/modules/food_and_drinks/food/snacks_other.dm [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+	name = "neep tatty haggis"
+	desc = "Oi mate! No neeps, but double beets! SCAM!!!"
+	icon_state = "neep_tatty_haggis"
+	icon = 'hippiestation/icons/obj/food/food.dmi'
+	bonus_reagents = list("nutriment" = 20, "vitamin" = 10, "iron" = 10)
+	trash = /obj/item/trash/plate
+	foodtype = GRAIN | VEGETABLES | MEAT | GROSS
+
+/obj/item/reagent_containers/food/snacks/taco/leg
+	desc = "An untraditional taco with leg,cheese, and lettuce."
```
Comment (KayeArray) [inline hippiestation/code/modules/food_and_drinks/food/snacks_other.dm]: space between commas please

File: hippiestation/code/modules/food_and_drinks/food/snacks_burgers.dm [hunk 4]
```diff
@@ -0,0 +1,8 @@
+/obj/item/reagent_containers/food/snacks/burger/assburger
+	name = "assburger"
+	desc = "What the hell, thats not an domesticated donkey meat, its a literal buttburger!"
```
Comment (KayeArray) [inline hippiestation/code/modules/food_and_drinks/food/snacks_burgers.dm]: that's not domesticated donkey meat, it's a literal buttburger!

File: hippiestation/code/modules/food_and_drinks/food/snacks_bread.dm [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+	desc = "A chunk of delicious(?) haggis."
+	icon = 'hippiestation/icons/obj/food/food.dmi'
+	icon_state = "haggis_chunk"
+	list_reagents = list("nutriment" = 10, "vitamin" = 5)
+	trash = /obj/item/trash/plate
+	foodtype = MEAT | GROSS | GRAIN
+
+/obj/item/reagent_containers/food/snacks/hearthytoast
+	name = "hearthy toast"
+	desc = "A human hearts on a buttered toast, really?"
```
Comment (KayeArray) [inline hippiestation/code/modules/food_and_drinks/food/snacks_bread.dm]: A human heart on buttered toast, really?

File: hippiestation/code/modules/food_and_drinks/food/snacks_bread.dm [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+	name = "haggis chunk"
+	desc = "A chunk of delicious(?) haggis."
+	icon = 'hippiestation/icons/obj/food/food.dmi'
+	icon_state = "haggis_chunk"
+	list_reagents = list("nutriment" = 10, "vitamin" = 5)
+	trash = /obj/item/trash/plate
+	foodtype = MEAT | GROSS | GRAIN
+
+/obj/item/reagent_containers/food/snacks/hearthytoast
+	name = "hearthy toast"
```
Comment (KayeArray) [inline hippiestation/code/modules/food_and_drinks/food/snacks_bread.dm]: *hearty

File: hippiestation/code/modules/food_and_drinks/food/snacks_pizza.dm [hunk 7]
```diff
@@ -0,0 +1,18 @@
+/obj/item/reagent_containers/food/snacks/pizza/cornpotato
+	name = "cornpotato-pizza"
+	desc = "The most disgusting pizza in galaxy."
```
Comment (KayeArray) [inline hippiestation/code/modules/food_and_drinks/food/snacks_pizza.dm]: A sanity destroying other thing

File: hippiestation/code/modules/food_and_drinks/food/snacks_pizza.dm [hunk 8]
```diff
@@ -0,0 +1,18 @@
...
+	icon = 'hippiestation/icons/obj/food/pizzaspaghetti.dmi'
+	icon_state = "pizzacornpotato"
+	slice_path = /obj/item/reagent_containers/food/snacks/pizzaslice/cornpotato
+	bonus_reagents = list("nutriment" = 8, "vitamin" = 8)
```
Comment (KayeArray) [inline hippiestation/code/modules/food_and_drinks/food/snacks_pizza.dm]: no toxins?

**Comment by Pyko1** [comment]
fixed things that bantz had a bone with

**Comment by bardofrage** [comment]
Plus uno

**Pull request merged**